omniplus/__init__.py

```diff
@@ -1,3 +1,3 @@
-__version__ = '0.0.1'
+__version__ = '0.0.2'
```

Comment:
Understood! Please provide the code you'd like me to review, and I will analyze it based on the specified focus areas and provide feedback in the requested format.